code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -121,7 +121,7 @@
 /obj/item/weapon/storage/firstaid/tactical/New()
 	..()
 	if (empty) return
-	new /obj/item/clothing/tie/stethoscope( src )
+	new /obj/item/clothing/accessory/stethoscope( src )
 	new /obj/item/weapon/surgicaldrill(src)

```

Comment:
the surgical drill is hotline miami as fuck, better to remove the stetho since it serves no real purpose